packages/dendron-plugin-views/src/components/DendronSchemaPage.tsx

```diff
@@ -0,0 +1,5 @@
+import { DendronProps } from "../types";
+
+export default function DendronSchemaPage({}: DendronProps) {
```

Comment:
did you get this to render?

Reply:
Had to add it to `VALID_NAMES` - will push soon!